clientd3d/music.c

```diff
@@ -43,9 +47,10 @@ static BYTE *pMIDIImmediate;
 #endif
 
 enum {SOUND_MIDI, SOUND_MUSIC};
+enum {LOAD_MUSIC, LOAD_MIDI}; /* Arguments for OpenMidiFile MCI device naming */
```

Comment:
The more readable way to do this is:
```
typedef enum {LOAD_MUSIC, LOAD_MIDI} LoadMusicType;

DWORD OpenMidiFile(const char *lpszMIDIFileName, LoadMusicType musicType)
```

This way, it's clear to the caller that they're supposed to pass in a value from the enum, and not any old integer.

Although you should use a better name than "LoadMusicType", since one of the values is "LOAD_MUSIC", which doesn't make much sense.

Reply:
Thanks.  Done.